webrender/src/resource_cache.rs

```diff
@@ -651,6 +674,87 @@ impl ResourceCache {
             }
         }
     }
+    fn update_texture_cache(cached_images: &mut ResourceClassCache<ImageRequest, CachedImageInfo>,
```

Comment:
nit: may look better if it's not static
What borrow problems did you encounter? Maybe we can work around them in a cleaner way.